dts/bindings/sensor/kionix,kx022.yaml

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) 2021 G-Technologies Sdn. Bhd.
+# SPDX-License-Identifier: Apache-2.0
+
+description: KX022 16-Bit digital I2C Accelerometer sensor
+
+compatible: "kionix,kx022"
+
+include: i2c-device.yaml
+properties:
+    int-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The INT1 pin
+
+    int-pin-1-polarity:
+      type: int
+      required: false
+      default: 1
+      enum:
+        - 0
+        - 1
+      description: |
+        Polarity of Physical Interrupt Pin 1
+        0: active low
+        1: Active high
```

Comment:
Is this the same pin as `INT1`? If so, then why isn't this information already encoded in the flags field of the `int-gpios` peroperty?